### PR TITLE
ENH: Add tests for size check for PR

### PR DESCRIPTION
**Pull request by souravsingh**
This Pull Request adds a test to check for size for Pull Requests.
Fixes https://github.com/scikit-beam/scikit-beam/issues/420

### CODE REVIEW

**Comment by tacaswell** [comment]
'power cycled' the PR to try and get travis going.

File: .travis.yml [hunk 1]
```diff
@@ -51,10 +51,18 @@ script:
       ./build_docs.sh;
     fi;
 
+ # Check repo size does not expand too much
+    - if [ "${TEST}" == "extra" ]; then
```
Comment (ericdill) [inline .travis.yml]: In pyqtgraph, a `TEST` environmental variable is defined.  We are not defining such an environmental variable but we could use the `BUILD_DOCS` environmental variable as proxy. Change this line to be 

`- if [ "$BUILD_DOCS" == "true" ]; then`

**Comment by ericdill** [comment]
You will also need to add a `before_script` section with most of this if block: https://github.com/pyqtgraph/pyqtgraph/blob/develop/.travis.yml#L115

**Comment by ericdill** [comment]
`start_test` is a bash function that is defined in the pyqtgraph .travis.yml that is not defined in ours.  Can you just remove the two lines that contain `start_test ...`?

**Comment by ericdill** [comment]
Same with `check_output`

File: .travis.yml [hunk 2]
```diff
@@ -38,11 +38,34 @@ install:
   - python setup.py install build_ext -i
   - pip install codecov
 
+before_script:
+- if [ "${TEST}" == "extra" ]; then
+        mkdir ~/repo-clone && cd ~/repo-clone &&
```
Comment (ericdill) [inline .travis.yml]: add these two lines at the beginning of this if block: https://github.com/pyqtgraph/pyqtgraph/blob/develop/.travis.yml#L36-L37

**Comment by ericdill** [comment]
Ok, this is way more tedious than I thought it was going to be when I made #420 . Thank you for tackling this!

**Comment by souravsingh** [comment]
Is there any other change I need to make?

**Comment by tacaswell** [comment]
This is failing on the doc build test https://travis-ci.org/scikit-beam/scikit-beam/jobs/121410831#L822

**Comment by ericdill** [comment]
@tacaswell this is actually still failing on the repo size check.  Let me have another look and give some guidance

**Comment by ericdill** [comment]
Err, meant to leave that comment on this PR...

( Copied from #420 )

Thanks for pushing this along @souravsingh .  I made some changes and tested on travis-ci for the ericdill/scikit-beam fork.  This PR got closed automatically, as I merged locally and pushed to master (normally don't do this, but I rebased a few times to get rid of the extra merge commits in this PR).  Thanks!  (note that your commits other than the merge commits are in master now: https://github.com/scikit-beam/scikit-beam/commits/master)  :smile:

**Comment by souravsingh** [comment]
Thanks for merging the PR @ericdill

**Pull request closed without merging**